.github/workflows/precommit.yml

```diff
@@ -382,8 +382,9 @@ jobs:
         # only run doctest for macos on aws
       - if: ${{ matrix.os == 'macos-latest' && matrix.cloud-provider == 'aws' }}
         name: Run Snowpark pandas API doctests
-        run: python -m tox -e "py${PYTHON_VERSION}-doctest-snowparkpandasdoctest-modin-ci"
+        run: python -m tox -e "modin_pandas_version-py${PYTHON_VERSION}-doctest-snowparkpandasdoctest-modin-ci"
         env:
+          MODIN_PANDAS_PATCH_VERSION: 2.2.2  # TODO unpin when 2.2.3 is available in snowflake anaconda
```

Comment:
why do we need to pin the pandas version here, is it because we haven't tested with 2.2.3 yet?

Reply:
as above--not pinning makes UDF/sprocs fail